src/components/Sections/ItemsSection.js

```diff
@@ -11,6 +11,7 @@ import { sortByFieldsWithPriority } from '../../utils/sort';
 
 const DESCRIPTION_KEY = 'description';
 const SECTION_ITEM_PADDING = 5;
+const RERENDER_TIMEOUT_MS = 550;
```

Comment:
@yonatan-roth I got to 550ms as the minimum timeout required to render it w/o textoverflowing, but as you might figure - this is a total guess